pytests/conftest.py

```diff
@@ -0,0 +1,244 @@
+from tblib import pickling_support
+
+pickling_support.install()
+
+import contextlib
+import functools
+import multiprocessing as mp
+import os
+import posixpath
+import signal
+import socket
+import sys
+import tempfile
+from six import reraise as raise_
+from six.moves.urllib.parse import quote, urljoin, urlunsplit
+
+import pytest
+import requests
+import requests_unixsocket
+
+import bjoern
+
+
+def free_port():
+    with contextlib.closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
+        s.bind(('', 0))
+        s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
I guess this might break on systems that don't support reuseaddr?

Reply:
Indeed, `socket.REUSEADDR` should be handled same as `socket.REUSEPORT`.